documentation/01-Prerequisites.md

```diff
@@ -111,6 +81,113 @@ sudo apt-get install exuberant-ctags
 
 > When available `make ctags` can be invoked to help generate a `tags` database at the root of the source tree to allow easier code navigation in editors that support it.
 
+#### Docker

```

Comment:
Just above here, there is a path error:
`./script/...` should be `./scripts/...`

Do you want to fix that while you're in here or shall I make a separate PR?